### DISCUSSION TITLE
[D] An error occurred when trying to access the custom event folder.

### DISCUSSION THREAD

**Discussion by SKIQQ-git** [dialogic-godot/dialogic#1238] [Q&A]
guys, i'm making an rpg game and i get this error when i export it to windows i get this error **[D] An error occurred when trying to access the custom event folder.** I don't know how to solve this error, could you help me?

**Comment by digitpdigit** [comment]
Hey, have this issue solved yet?

**Reply by Jowan-Spooner** [reply]
Hmmm, not sure. Check out #942 maybe?
